applications/launchpad/gui-react/src/components/CoinsList/index.tsx

```diff
@@ -19,7 +19,10 @@ const CoinsList = ({ coins, color }: CoinsListProps) => {
   return (
     <StyledCoinsList color={color}>
       {coins.map((c, idx) => (
-        <CoinsListItem key={`coin-${idx}`} loading={c.loading}>
+        <CoinsListItem
+          key={`coin-${idx}`}
+          loading={c.loading ? c.loading.toString() : 'false'}
```

Comment:
I'm strongly against using string for boolean props

Reply:
Switching to `$loading`